client/starwhale/api/_impl/data_store.py

```diff
@@ -646,6 +648,7 @@ def __init__(
                 self.columns = columns
                 self.explicit_none = explicit_none
 
+        logger.debug("scan enter, table size:{}", len(tables))
```

Comment:
`logger.debug("scan enter, table size:{}", len(tables))` -> `logger.debug(f"scan enter, table size:{len(tables)}")` ?

Reply:
688, 695 line meet the some issue.